static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -29,12 +31,26 @@ interface Props {
 
 interface TabSettings {
   title: string
-  id: TabIds
-  contentType: ContentType
+  id: string
+  contentType: string
   resourcetype: string | null
   embeddable: boolean
   singleColumn: boolean
-  sourceWebsiteName?: string
+}
+
+export function toTabSettings(
```

Comment:
**Request:** don't export this, it's not used anywhere else. Sorry if I missed that in initial review.

**Very Minor Suggestion:** Personally, I prefer [arrow function expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), i.e., 
```ts
const toTabSettings = (contentType: string, contentTitle: string) => {...}
```
The primary differences are:
- functions declared with `function` keyword are [hoisted](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting)
- functions declared with the `function` keyword have their own `this` context, whereas arrow functions **always** use the `this` context in which they were defined. .... [`this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) in js kinda quirky.

Both of the things above sometimes cause confusion... so unless I really need a function with its own `this` context, I tend to avoid functions declared with function keyword.

Reply:
Updated in a1f7c9758490ba9de2953f293f058057e134db9f.